detection_rules/attack.py

```diff
@@ -5,20 +5,23 @@
 """Mitre attack info."""
 
 import json
-import os
+from pathlib import Path
 import requests
 from collections import OrderedDict
 
 from .semver import Version
-from .utils import get_etc_path, get_etc_glob_path, read_gzip, gzip_compress
+from .utils import read_gzip, gzip_compress
+
+ROOT_DIR = Path(__file__).parent.parent
+ETC_DIR = ROOT_DIR / 'etc'
```

Comment:
```suggestion
from .utils import ETC_DIR, read_gzip, gzip_compress
```

I would just import these from utils as needed for all these files

Reply:
Done. Thanks!